dll/win32/shell32/lang/pl-PL.rc

```diff
@@ -814,35 +814,35 @@ BEGIN
     IDS_OVERWRITEFILE_CAPTION "Potwierdź zastąpienie pliku"
     IDS_OVERWRITEFOLDER_TEXT "Ten folder zawiera już katalog o nazwie '%1'.\n\nJeżeli w docelowym folderze wystąpią pliki o takich samych nazwach jak\nw wybranym folderze, to zostaną one zastąpione. Czy chcesz mimo to przenieść\nlub skopiować folder?"
 
-    IDS_FILEOOP_COPYING "Kopiowanie..."
-    IDS_FILEOOP_MOVING "Przenoszenie..."
-    IDS_FILEOOP_DELETING "Usuwanie..."
+    IDS_FILEOOP_COPYING "Trwa kopiowanie..."
+    IDS_FILEOOP_MOVING "Trwa przenoszenie..."
+    IDS_FILEOOP_DELETING "Trwa usuwanie..."
     IDS_FILEOOP_FROM_TO "Z %1 do %2"
     IDS_FILEOOP_FROM "Z %1"
-    IDS_FILEOOP_PREFLIGHT "Przygotowywanie"
+    IDS_FILEOOP_PREFLIGHT "Przygotowywanie operacji"
 
     /* message box strings */
     IDS_RESTART_TITLE "Uruchom ponownie"
-    IDS_RESTART_PROMPT "Czy chcesz ponownie uruchomić system?"
-    IDS_SHUTDOWN_TITLE "Zamknij"
-    IDS_SHUTDOWN_PROMPT "Czy chcesz zamknąć system?"
+    IDS_RESTART_PROMPT "System ReactOS zamyka się i uruchamia ponownie"
```

Comment:
Incorrect translation (has different meaning than the english resource). Please revert to previous version.

Reply:
is correct :)

Reply:
No it isn't, unless the english resource is wrong there. If that's a case, then please fix it first and then update translations accordingly.